dataflow/src/main/java/org/checkerframework/dataflow/cfg/builder/CFGTranslationPhaseOne.java

```diff
@@ -1298,6 +1319,26 @@ protected List<Node> convertCallArguments(
     int numFormals = formals.size();
 
     ArrayList<Node> convertedNodes = new ArrayList<>(numFormals);
+    AnnotationMirror assertMethodAnno =
+        annotationProvider.getDeclAnnotation(method, AssertMethod.class);
+    int param = -1;
+    TypeMirror assertType = null;
+    boolean assertTrue = true;
```

Comment:
To avoid re-assignment (and, more importantly, make clear that there is no reassignment), I would declare these variables where they currently are declared, but remove the initializers and put them in an `else` clause.